src/controllers/module.controller.js

```diff
@@ -92,20 +92,22 @@ const updateModuleById = async (req, res, next) => {
         }
       )
 
-      updatedModule.addActions(action_id)
       if (updatedModule.length) {
```

Comment:
Aqui deberia usarse count y no updatedModule.length porque updatedModule siempre sera un array con algun item.
```suggestion
      if (count) {
```

Reply:
Ok